modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -61,7 +61,7 @@ mirror:
 <2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
 <3> Set the registry URL for the storage backend.
 <4> Set the channel to retrieve the {product-title} images from.
-<5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for an improved cluster update experience when using the web console. For more information, see _About the OpenShift Update Service_.
+<5> Add `graph: true` to build and push the graph image to the mirror registry. The `graph: true` option also generates the `UpdateService` CR manifest that the OpenShift Update Service (OSUS) uses to provide a graph image for an improved cluster update experience when using the web console and command-line interface (CLI) in a disconnected environment. For more information, see _About the OpenShift Update Service_.
```

Comment:
@PratikMahajan could you please suggest on the wording?

I think it's enough to mention:
Add `graph: true` to build and push the graph-data image to the mirror registry. The graph-data image is required to create OpenShift Update Service (OSUS)  OSUS application. The option also generates the `UpdateService` CR manifest that can be directly used by oc command-line interface (CLI) to create OpenShift Update Service (OSUS)  OSUS application.

Reply:
@shellyyang1989's suggestion accurately tells what oc-mirror is supposed to do. 
We should mention that the CR created by oc-mirror can be directly used by CLI. On the web-console front, the CR can be created using the form or we can directly apply the yaml. But form seems to be the default and easier to use on web console. we dont need to mention the form part in the docs. 
Lets go with @shellyyang1989's wording